src/frontend/src/lib/utils/token.utils.ts

```diff
@@ -0,0 +1,47 @@
+import { E8S_PER_ICP } from '$lib/constants/icp.constants';
```

Comment:
This is unrelated to this new `SendInputAmount` cmp right? Can you provide it in a separate PR.